### PR TITLE
[Feature] Add current catalog name to AuditEvent

### PR DESCRIPTION
**Pull request by stephen-shelby**
## What type of PR is this：
- [ ] bug
- [x] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 9 / 9 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/SqlTaskRunProcessor.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/scheduler/PartitionBasedMaterializedViewRefreshProcessor.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/plugin/AuditEvent.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/ConnectProcessor.java|2|2|100.00%|[]|

**Review by DorianZheng — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMaterializedViewRefreshProcessor.java [hunk 1]
```diff
@@ -392,7 +392,8 @@ private InsertStmt generateInsertStmt(Set<String> materializedViewPartitions,
                 .setTimestamp(System.currentTimeMillis())
                 .setClientIp(mvContext.getRemoteIp())
                 .setUser(ctx.getQualifiedUser())
-                .setDb(ctx.getDatabase());
+                .setDb(ctx.getDatabase())
+                .setCatalog(ctx.getCurrentCatalog());
```
Comment (chaoyli) [inline fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMaterializedViewRefreshProcessor.java:396]: Internal catalog is not need in the event.

Comment (stephen-shelby) [reply fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMaterializedViewRefreshProcessor.java]: done

**Review by chaoyli — APPROVED** [review]

**Comment by stephen-shelby** [comment]
run starrocks_fe_unittest

**Comment by stephen-shelby** [comment]
run starrocks_fe_unittest

**Review by dirtysalt — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by stephen-shelby** [comment]
run starrocks_fe_unittest

**Comment by stephen-shelby** [comment]
run starrocks_fe_unittest

**Comment by stephen-shelby** [comment]
run starrocks_fe_unittest

**Comment by stephen-shelby** [comment]
un starrocks_fe_unittest

**Comment by stephen-shelby** [comment]
run starrocks_fe_unittest

**Comment by stephen-shelby** [comment]
run starrocks_fe_unittest

**Comment by stephen-shelby** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 7 / 7 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/SqlTaskRunProcessor.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/plugin/AuditEvent.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/ConnectProcessor.java|2|2|100.00%|[]|

**Comment by stephen-shelby** [comment]
@Mergifyio backport branch-2.3

**Comment by stephen-shelby** [comment]
@Mergifyio backport branch-2.4

**Pull request merged**